examples/api/test/widgets/inherited_notifier/inherited_notifier.0_test.dart

```diff
@@ -0,0 +1,53 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/widgets/inherited_notifier/inherited_notifier.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('It rotate the spinners', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const example.InheritedNotifierExampleApp(),
+    );
+
+    final Iterable<Transform> widgets = tester.widgetList<Transform>(find.ancestor(of: find.text('Whee!'), matching: find.byType(Transform)));
+    expect(widgets, everyElement(isA<Transform>().having((Transform widget) => widget.transform, 'transform', Matrix4.identity())));
```

Comment:
To reduce verbosity, consider using an utility matcher, such as:

```suggestion
    Matcher transforms(Matrix4 matrix) {
      return everyElement(isA<Transform>().having((Transform widget) => widget.transform, 'transform', matrix));
    }
    
    expect(widgets, transforms(Matrix4.identity()));
```

Not sure about the name, maybe we can find a better one.

Reply:
What do you think of [refactor: Use a utils matcher](https://github.com/flutter/flutter/pull/150344/commits/3c94fc58d1e330406584ec34bb8249a87062b9db) ?